resources/core/charts/kubeless/charts/lambdas-ui/README.md

```diff
@@ -0,0 +1,123 @@
+# Lambdas
+
+## Overview
+
+**Lambdas** is a web-based UI component which is a part of the Console UI.
+It allows you to manage lambda functions, which are the key part of Kyma capabilities.
+
+## Details
+
+**Lambdas** allow you to:
+- Create and modify lambda functions.
+- Expose a function as API.
+- Bind a function to a service.
+- Write the code of the function. This option is available only through node.js.
+- Manage dependencies.
+- Add and remove labels.
+- Enable API security
+- Select function size.
+
+### Configuration
+
+ **Lambdas** are deployed with the use of a [ConfigMap](templates/configmap.yaml).
+The ConfigMap introduces a `config.js` file that is mounted as the asset of the Console application and injected as a configuration file.
+Use this mechanism to overwrite the default configuration with custom values resulting from the Helm chart installation.
+
+### Function size
+
+You can modify the size of a function when you either create or update a function in **Lambdas**.
+
+The set of values for a function size such as `resources:` (`requests:` and `limits:`) and `horizontalPodAutoscaler:` (`minReplicas:`, `maxReplicas:` and `targetAverageUtilization:`) are defined in the [ConfigMap](templates/configmap.yaml), explained in the previous section.
+
+
+- The initial **requests:** configuration for a function is as follows:
+
+```
+resources:
+  requests:
+    cpu: '100m'
+    memory: '100Mi'
+```
+
+- You can configure the sizes using the following parameters: 
+1. S:
+   * `{memory: '128Mi', cpu: '100m', minReplicas: 1, maxReplicas: 2}`
+2. M:
+   * `{memory: '256Mi', cpu: '500m', minReplicas: 2, maxReplicas: 5}`
+3. L: 
+   * `{memory: '512Mi', cpu: '1.0',  minReplicas: 5, maxReplicas: 10}`
+4. XL:
+   * `{memory: '1024Mi', cpu: '2.0', minReplicas: 10, maxReplicas: 20}`
+
```

Comment:
Maybe we can change the structure above to have the size and the configuration in one line, like this:

```
1. S: `{memory: '128Mi', cpu: '100m', minReplicas: 1, maxReplicas: 2}`
2. M: `{memory: '256Mi', cpu: '500m', minReplicas: 2, maxReplicas: 5}`
3. L: `{memory: '512Mi', cpu: '1.0',  minReplicas: 5, maxReplicas: 10}`
4. XL: `{memory: '1024Mi', cpu: '2.0', minReplicas: 10, maxReplicas: 20}`
```